spec/automation/unit/method_validation/vm_migrate_task_complete_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe 'vmmigratetask_complete method' do
+  let(:miq_server)       { EvmSpecHelper.local_miq_server }
+  let(:user)             { FactoryGirl.create(:user_with_group, :userid => 'admin', :email => "lufu@redhat.com") }
+  let(:miq_request_task) { FactoryGirl.create(:miq_request_task, :miq_request => request, :source => vm) }
+  let(:request)          { FactoryGirl.create(:vm_migrate_request, :requester => user) }
+
+  let(:tenant)           { Tenant.root_tenant }
+  let(:ems)              { FactoryGirl.create(:ems_vmware, :tenant => tenant) }
```

Comment:
No need for a separate let block for tenant.  Just change the ems to reference `:tenant => Tenant.root_tenant`